plenum/test/view_change/test_view_change_after_back_to_quorum_with_disconnected_primary.py

```diff
@@ -14,7 +14,6 @@ def tconf(tconf):
         yield tconf
 
 
-@pytest.mark.skip(reason="INDY-2223: Temporary skipped to create build")
 def test_view_change_after_back_to_quorum_with_disconnected_primary(txnPoolNodeSet, looper,
```

Comment:
I believe there are 2 tests with this name. Can you please make sure that both of them are unskipped and pass?

Reply:
Done.